_sass/landing/signup.scss

```diff
@@ -1,223 +1,34 @@
-    margin-bottom: 15px;
-
-    @include media(xxxs) {
-      margin-bottom: 9px;
-    }
-  }
-
-  .form {
-    overflow: visible;
-  }
-
-  .popover-trigger {
-    cursor: pointer;
-    margin-left: 3px;
-    top: -1px;
-  }
-
-  .popover {
-    color: $gray;
-    font-size: 14px;
-    line-height: 1.4;
-    width: 280px;
-  }
-
-  .input-group-vertical {
-    flex: 0 0 auto;
-    margin-top: 15px;
-    overflow: visible;
-
-    @include media(sm) {
-      min-width: 100%;
-
-      &:first-child {
-        padding-bottom: 12px;
-      }
-    }
-
-    .input-md {
-      border-radius: $input-border-radius-lg;
-    }
-  }
-
-  .radio-group {
-    position: relative;
-
-    > .grid-block {
-      @include media(xxs) {
-        flex-flow: column nowrap;
-      }
-    }
-
-    .p {
-      padding-bottom: 10px;
-
-      @media (max-width: 380px) {
-        margin: 0 auto;
-        max-width: 210px;
-      }
-    }
-  }
-
-  .input-radio {
-    bottom: 0;
-    opacity: 0;
-    pointer-events: none;
-    position: absolute;
-
-    // active state
-    &:checked + .radio,
-    &:checked + .radio:hover {
-      border-color: currentColor;
-      color: $purple-light;
-
-      .icons:last-child {
-        background: $green;
-        color: $white;
-      }
-    }
-
-    // except when invalid
-    &.invalid:not(:valid) + .radio,
-    &.invalid:not(:valid) + .radio:hover {
-      border-color: currentColor;
-      color: $red;
-    }
-
-    // disabled state
-    &[disabled] {
-
-      &:not(:checked), {
-
-        + .radio,
-        + .radio:hover,
-        + .radio:active {
-          border-color: $gray-lighter;
-          color: $gray-dark;
-        }
-      }
-
-      + .radio {
-        cursor: not-allowed;
-      }
-    }
-  }
-
-  .radio {
-    align-items: center;
-    display: flex;
-    flex: 1 1;
-    height: 90px;
-    justify-content: center;
-    line-height: 86px;
-    padding-left: 0;
-    padding-right: 30px;
-
-    @include media(xs) {
-      font-size: 18px;
-    }
-
-    @include media(xxs) {
-      line-height: 56px;
-      max-height: 60px; // firefox
-    }
-
-    &:last-child {
-      margin-left: 15px;
-
-      @include media(xxs) {
-        margin: 15px 0 0;
-      }
-    }
-
-    // logo
-    .icons:first-child {
-      height: 100%;
-      margin-right: 9px;
-      width: 24px;
-
-      @include media(xs) {
-        margin-right: 6px;
-        width: 21px;
-      }
-    }
-
-    // check
-    .icons:last-child {
-      border: 1px solid currentColor;
-      border-radius: 50%;
-      color: $gray-lighter;
-      height: 24px;
-      padding: 6px;
-      position: absolute;
-      right: 15px;
-      top: 31px;
-      width: 24px;
-
-      @include media(xxs) {
-        top: 16px;
-      }
-    }
-  }
-
-  // submit
-  .btn.green {
-    margin-top: 30px;
-
-    .spinner-wrapper {
-      height: 100%;
-      padding: 9px;
-      position: absolute;
-      right: 6px;
-      top: 0;
-      width: 39px;
-    }
   }
 }
```

Comment:
I've deleted this file in master, so just recommit your change after merging and only accept your changes.